### PR TITLE
Some testplan adjustments

### PR DESCRIPTION
**Pull request by ldoktor**
First 2 commits are minor adjustments, the last 2 commits are probably controversial but I don't see a point in spending time on those projects, unless we decide to re-open them again.

### CODE REVIEW

File: examples/testplans/release.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
 	 "description": "On your development machine, build the Avocado RPM packages using: `$ make rpm`. Expected result: SRPM and RPM files at `BUILD/RPM/python-avocado-x.y.z-r.distro.{src,noarch}.rpm`"},
 
 	{"name": "Avocado RPM lint",
-	 "description": "With the built RPMs, check if there are no packaging errors or warnings by running: `$ rpmlint BUILD/RPM/python-avocado-x.y.z-r.distro.noarch.rpm`.  Expected result: 1 packages and 0 specfiles checked; 0 errors, 0 warnings."},
+	 "description": "With the built RPMs, check if there are no unexpected packaging errors or warnings by running: `$ rpmlint BUILD/RPM/python*-avocado-x.y*.rpm`.  Expected result: 3 packages and 0 specfiles checked; 0 errors, 0 warnings. (some missing man-pages can be ignored)"},
```
Comment (clebergnu) [inline examples/testplans/release.json]: OK, but the commit message has a typo ("úy2/py3").

File: examples/testplans/release.json [hunk 2]
```diff
@@ -39,21 +39,6 @@
 	{"name": "Avocado Server Functional Test",
 	 "description": "On the same machine used during 'Avocado Server Run', on another console, run `$ curl http://localhost:9405`. Expected result: json response"},
 
-	{"name": "Avocado Virt and VT Source Checkout",
```
Comment (clebergnu) [inline examples/testplans/release.json]: Agreed.

File: examples/testplans/release.json [hunk 3]
```diff
@@ -30,12 +30,6 @@
 	{"name": "Avocado Remote Machine HTML report",
 	 "description": "On a web browser, open the previously generated  HTML report at `/tmp/report.html`. Verify that all the links such as `job-YYYY-MM-DD...` (under `Results Dir`), `1-/**/avocado/tests/**/examples/tests/passtest.py:PassTest.test` (under `Test ID`) and `debug.log` point to valid locations."},
 
-	{"name": "Avocado Server Source Checkout and Unittests",
```
Comment (clebergnu) [inline examples/testplans/release.json]: OK.

**Comment by ldoktor** [comment]
OK, force-updated the commit message (and rebased)

**Pull request merged**